pandas/src/parser/tokenizer.c

```diff
@@ -1675,23 +1483,20 @@ double xstrtod(const char *str, char **endptr, char decimal,
   num_digits = 0;
   num_decimals = 0;
 
-  // Process string of digits
-  while (isdigit(*p))
-  {
+  // Process string of digits.
```

Comment:
Because the checker is in Python.  Also, that's inconvenient for people who are using Python 3 for development and / or personal use.